Bottom: 315804cd6a6565ba6da07a6118651622ed467ff2
Top:    171444649d08ba68113e9e1ae7dab9901d7b9e09
Author: Jason Wang <jasowang@redhat.com>
Date:   2018-07-04 12:07:51 +0800

Refresh of vhost-support-packed-ring

---

diff --git a/drivers/vhost/vhost.h b/drivers/vhost/vhost.h
index d071daf..0f80b62 100644
--- a/drivers/vhost/vhost.h
+++ b/drivers/vhost/vhost.h
@@ -264,7 +264,8 @@ enum {
 			 (1ULL << VIRTIO_RING_F_EVENT_IDX) |
 			 (1ULL << VHOST_F_LOG_ALL) |
 			 (1ULL << VIRTIO_F_ANY_LAYOUT) |
-			 (1ULL << VIRTIO_F_VERSION_1)
+			 (1ULL << VIRTIO_F_VERSION_1) |
+			 (1ULL << VIRTIO_F_RING_PACKED)
 };
 
 static inline bool vhost_has_feature(struct vhost_virtqueue *vq, int bit)
